runtime/index.d.ts

```diff
@@ -1,28 +1,53 @@
-declare module "@sapper/app"
-declare module "@sapper/server"
-declare module "@sapper/service-worker"
+declare module "@sapper/app";
+declare module "@sapper/server";
+declare module "@sapper/service-worker";
+
+interface Stores {
+	preloading: {
+		subscribe(preloading: boolean): () => void;
```

Comment:
There are ready-made types for Stores in svelte that you could reuse that give a more complete type definition for subscribe (there's an optional second parameter and a return value that are missing here):

I think if you just do

```
import { Readable, Writable } from 'svelte/store';

...
	preloading: Readable<boolean>;
```

you should get the correct definitions for `subscribe` etc for free.

For the page store, there is also already a `PageContext` interface, so you could declare it as

```
	import { PageContext } from '@sapper/app/types';
...
	page: Writeable<PageContext>;
```

Reply:
Done